src/content/blog/astro-440.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: "Astro 4.4"
+description: "Astro 4.4 is now available! This release includes the addition of performance audits for the dev toolbar, performance upgrades, the ability to automatically infer the dimensions of remote images, and more."
+publishDate: "February 15, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-440/post-header-4.4.webp"
+socialImage: "/src/content/blog/_images/astro-440/og-image-4.4.webp"
+lang: "en"
+---
+
+import perfAuditImage from "./_images/astro-440/perf-audit.webp"
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+
+Astro 4.4 is now available! This release includes performance audits for the dev toolbar, performance upgrades, the ability to automatically infer the dimensions of remote images, and more.
```

Comment:
A bit of a nit but “performance upgrades” seemed a bit vague. Reusing the heading like this could be a bit clearer?

```suggestion
Astro 4.4 is now available! This release includes performance audits for the dev toolbar, improved streaming performance, the ability to automatically infer the dimensions of remote images, and more.
```